.travis/script.sh

```diff
@@ -39,11 +39,21 @@ case "${BUILD_TARGET}" in
         ;;
 
     linux-auto-gcc-check-happy)
+        ls /tmp/happy_debug*
+        mkdir -p $TRAVIS_BUILD_DIR/happy-test-logs/linux-auto-gcc-check-happy/from-tmp
         sudo make -f Makefile-Standalone DEBUG=1 TIMESTAMP=1 COVERAGE=1 BuildJobs=24 check
+        cp $TRAVIS_BUILD_DIR/build/x86_64-unknown-linux-gnu/src/test-apps/happy $TRAVIS_BUILD_DIR/happy-test-logs/linux-auto-gcc-check-happy -rf
+        cp /tmp/happy* $TRAVIS_BUILD_DIR/happy-test-logs/linux-auto-gcc-check-happy/from-tmp
+        echo "please check happy-test-log/<build_number> under link: https://storage.cloud.google.com/openweave"
         ;;
 
     linux-lwip-gcc-check-happy)
+        ls /tmp/happy_debug*
+        mkdir -p $TRAVIS_BUILD_DIR/happy-test-logs/linux-lwip-gcc-check-happy/from-tmp
```

Comment:
This block is already switching on BUILD_TARGET, so there's no need to copy-and-paste the literal value of BUILD_TARGET, just reference the variable.